packages/rest/src/http/decorators.ts

```diff
@@ -563,17 +563,24 @@ export function getAuthentication(
  *
  * `@route` can only be applied to operations, namespaces, and interfaces.
  */
-export function $route(context: DecoratorContext, entity: Type, path: string) {
+export function $route(context: DecoratorContext, entity: Type, path: string, parameters?: Model) {
   setRoute(context, entity, {
     path,
     isReset: false,
+    parameters,
   });
 }
 
-export function $routeReset(context: DecoratorContext, entity: Type, path: string) {
+export function $routeReset(
```

Comment:
We really need to remove this decorator, it actually does nothing :)  That a change for another PR, though.